src/pages/UsersPage.tsx

```diff
@@ -0,0 +1,56 @@
+import { useEffect, useState } from "react";
+import { getRegisteredUsers } from "../api/services/users";
+import { SearchLayout } from "../components/Layout";
+import { toastError } from "../components/Toast/toast";
+import { UserList } from "../components/UserList";
+import { useLoader } from "../hooks";
+
+export function UsersPage() {
+  const [users, setUsers] = useState<any>([]);
+  const { loading, setLoading } = useLoader();
+
+  async function fetchUsersList() {
+    setLoading(true);
+    try {
+      const { data } = await getRegisteredUsers();
+      const usersList = data?.map((item) => {
+        return {
+          id: item.id,
+          created_at: new Date(item.created_at),
+          state: item.state,
+          state_changed_at: new Date(item.state_changed_at),
+          updated_at: new Date(item.updated_at),
+          name: `${item.traits?.name?.first ?? ""} ${
+            item.traits?.name?.last ?? ""
+          }`,
+          email: item.traits.email
```

Comment:
Shouldn't this be done in getRegisteredUsers() ?

Reply:
it depends, if there is a chance of using the same format data in multiple places then we can move it the getRegisteredUsers.

Reply:
It isn't so much whether it will be used in multiple places, but a separation of data / server handling and UI.